llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -1473,6 +1473,13 @@ def FeatureWaitsBeforeSystemScopeStores : SubtargetFeature<
   "Target requires waits for loads and atomics before system scope stores"
 >;
 
+def FeaturePKF32Insts : SubtargetFeature<"pk-f32-insts",
```

Comment:
There is already FeaturePackedFP32Ops. The name of feature is misleading at least. Then you probably only need a check in the subtarget anyway, not a feature itself.